packages/core/src/Language.ts

```diff
@@ -0,0 +1,15 @@
+import * as t from 'io-ts';
+
+export const Language = t.union([
+  t.literal('TypeScript'),
+  t.literal('JavaScript'),
+  t.literal('Haskell'),
+  t.literal('Java'),
+  t.literal('C++'),
+  t.literal('C#'),
+  t.literal('F#'),
+  t.literal('Go'),
+  t.literal('C'),
+]);
```

Comment:
I don't know if it's a good idea or possible to use some abstraction here by mapping over the array of languages?
I will reduce some boilerplate.

```suggestion
import languags from 'somewhere'
export const Language = t.union(langauges.map(lang => t.literal(lang)));
```

I think the `languages` array is a useful thing and we can reuse it some where.

Reply:
hmm 
where do you suggest we get the languages from?
isn't it better to be in the same file?

Reply:
Yes, we can export that also.